src/modules/0.0.22/iasql_functions/iasql.ts

```diff
@@ -1019,7 +1019,9 @@ export async function continueUpgrade(
     INSERT INTO aws_credentials (access_key_id, secret_access_key)
     VALUES ('${creds.access_key_id}', '${creds.secret_access_key}');
   `);
+  const orm = context.orm;
   await commit(dbId, false, context, true);
+  context.orm = orm;
```

Comment:
How do these two lines do anything?

Reply:
because inside the `commit` fn I re-assign the `context.orm` with the inner `orm` object created, then that connection is closed and here it would fail to continue the upgrade.

Reply:
Got it. Can we add a comment to that effect? Or perhaps push that behavior into the `commit` function instead of requiring the callers of the function to replicate this logic?